Dockerfile

```diff
@@ -15,7 +15,6 @@ RUN poetry install
 
 FROM base AS production
 # COPY across Config
-COPY env_vars.py env_vars.py
```

Comment:
Definitely not exercise 3 :)

Reply:
That is true! Though `env_vars.py` is no longer needed by docker (and I've just realised it's not needed by anything else either! So I'll delete it).